.travis.yml

```
@@ -0,0 +1,16 @@
+sudo: false
+language: python
+dist: xenial
+virtualenv:
+  system_site_packages: true
+python:
+  - '3.5'
+  - '2.7'
+install:
+  - pip install flake8
+  - sudo apt-get install -y docbook-utils python-rpm python3-rpm rpm
```

this has `sudo: false` in the first line, and still works, nice.

how about
```suggestion
addons:
  apt:
    packages:
    - docbook-utils
    - python-rpm
    - python3-rpm
    - rpm
```

---

sudo is deprecated and now you always have sudo. That said, I still prefer the explicit install via addons.